components/pd_client/src/util.rs

```diff
@@ -848,11 +848,12 @@ pub fn check_resp_header(header: &ResponseHeader) -> Result<()> {
         ErrorType::IncompatibleVersion => Err(Error::Incompatible),
         ErrorType::StoreTombstone => Err(Error::StoreTombstone(err.get_message().to_owned())),
         ErrorType::RegionNotFound => Err(Error::RegionNotFound(vec![])),
-        ErrorType::Unknown => Err(box_err!(err.get_message())),
         ErrorType::GlobalConfigNotFound => {
             Err(Error::GlobalConfigNotFound(err.get_message().to_owned()))
         }
         ErrorType::Ok => Ok(()),
+        ErrorType::DuplicatedEntry | ErrorType::EntryNotFound => Err(box_err!(err.get_message())),
```

Comment:
ErrorType::DuplicatedEntry and ErrorType::EntryNotFound in kvproto are added by you?

Reply:
No, it's added by others but not yet used.

Reply:
No, but this commit was earlier than we wanted it to be, so I had to improvise an easy solution or the build wouldn't pass. @sticnarf did the same thing in this [pr](https://github.com/tikv/tikv/pull/13159)
<img width="1318" alt="image" src="https://user-images.githubusercontent.com/32640567/185533897-64f89637-d42d-4184-a49d-97b86b31f75b.png">